website/docs/d/pi_cloud_connection.html.markdown

```diff
@@ -37,28 +32,26 @@ data "ibm_pi_cloud_connection" "example" {
   ```
 
 ## Argument reference
-
 Review the argument references that you can specify for your data source.
 
 - `pi_cloud_instance_id` - (Required, String) The GUID of the service instance associated with an account.
 - `pi_cloud_connection_name` - (Required, String) The cloud connection name to be used.
 
 ## Attribute reference
-
 In addition to all argument reference list, you can access the following attribute references after your data source is created.
 
-- `id` - (String) The unique identifier of the cloud connection.
-- `classic_enabled` - (Bool) Is classic endpoint destination enabled?
+- `classic_enabled` - (Bool) Enable classic endpoint destination.
 - `connection_mode` - (String) Type of service the gateway is attached to.
-- `global_routing` - (String) Is global routing enabled for this cloud connection.
-- `gre_destination_address` - (String) The GRE destination IP address.
-- `gre_source_address` - (String) The GRE auto-assigned source IP address.
+- `global_routing` - (String) Enable global routing for this cloud connection.
+- `gre_destination_address` - (String) GRE destination IP address.
+- `gre_source_address` - (String) GRE auto-assigned source IP address.
+- `id` - (String) The unique identifier of the cloud connection.
 - `ibm_ip_address` - (String) The IBM IP address.
-- `metered` - (String) Is metered enabled for this cloud connection.
-- `networks` - (Set of String) Set of Networks attached to this cloud connection.
+- `metered` - (String) Enable metering for this cloud connection.
+- `networks` - (Set) Set of Networks attached to this cloud connection.
 - `port` - (String) Port.
 - `speed` - (Integer) Speed of the cloud connection (speed in megabits per second).
 - `status` - (String) Link status.
 - `user_ip_address` - (String) User IP address.
-- `vpc_crns` - (Set of String) Set of VPCs attached to this cloud connection.
-- `vpc_enabled` - (Bool) Is VPC enabled for this cloud connection?
+- `vpc_crns` - (Set) Set of VPCs attached to this cloud connection.
+- `vpc_enabled` - (Bool) Enable VPC for this cloud connection.
```

Comment:
Change to (Boolean) to match other parts of the IBM terraform provider documentation.